verify-changed-files/README.md

```diff
@@ -0,0 +1,55 @@
+# Verify Changed Files Action
+
+A GitHub Action to check if specified files have changed between git refs and output the results.
+
+## Inputs
+
+| Name | Description | Required | Default |
+|------|-------------|----------|---------|
+| `files` | Files or patterns to check for changes | Yes | N/A |
```

Comment:
can you make files optional and make sure it is all files by default?

Reply:
Done